examples/step-44/CMakeLists.txt

```diff
@@ -34,6 +34,17 @@ IF(NOT ${deal.II_FOUND})
     )
 ENDIF()
 
+#
+# Are all dependencies fulfilled?
+#
+IF(NOT DEAL_II_WITH_CXX11)
+  MESSAGE(FATAL_ERROR "
+Error! The deal.II library found at ${DEAL_II_PATH} was not configured with
+    DEAL_II_WITH_CXX11 = ON
+One or all of these are OFF in your installation but are required for this tutorial step."
```

Comment:
oh, i see, so it is that way in quite a number of places with a single dependency. Well, i would probably adjust it here but leave other tutorials for another time.